pkg/driverbuilder/builder/builders.go

```diff
@@ -55,6 +43,14 @@ type Config struct {
 	*Build
 }
 
+func (c Config) ToDriverFullPath() string {
+	return path.Join(DriverDirectory, "build", "driver", fmt.Sprintf("%s.ko", c.DriverName))
```

Comment:
We expect the built artifact with cmake to be under `/tmp/driver/build/driver/$drivername.ko`.